tests/unit_tests/rigid_pendulum_poc/test_quaternions.cpp

```diff
@@ -236,4 +236,236 @@ TEST(QuaternionTest, GetRotationVectorFromNullQuaternion) {
     ASSERT_NEAR(std::get<2>(v), std::get<2>(expected), 1e-6);
 }
 
+TEST(QuaternionTest, WrapAngleToPi) {
```

Comment:
This should be many different tests (one for each angle).  If we ever refactor wrap_angle_to_pi and introduce a bug, this will make debugging a little easier.

Reply:
<img width="1099" alt="image" src="https://github.com/Exawind/openturbine/assets/59463700/ce746819-8295-4d30-8cfe-d617b2240e2a">

Just wondering if we need more specific information than this for debugging - it provides the exact line # where the assertion fails, along with the calculated and expected values. My opinion is to keep these helper function testing condensed if we can help it, but open to your thoughts.

Reply:
It does. But "assert" macros end the test at that failure, so we don't know if there were any other failures later in the test.  

Also, long tests are harder to understand as a new person reading them, and it's hard to tell at first glance that each grouping is fully independent.

Making each independent condition its own test is the simplest way to solve both of these problems

Reply:
Sounds reasonable to me, let's break them up.